backend/src/config/capabilityStatementResources.json

```diff
@@ -123,16 +123,6 @@
           "name": "package",
           "definition": "http://hl7.org/fhir/us/cqfmeasures/OperationDefinition/Library-package"
         },
-        {
```

Comment:
We might want to add support for FHIR `create`/`update` to the capability statement now

Reply:
I think they're already there? Unless I'm missing somethign